### PR TITLE
Bluetooth: Tester: Adjust bap ext adv options

### PR DESCRIPTION
**Pull request by Frodevan**
This change removes the BT_LE_PER_ADV_OPT_USE_TX_POWER option for extended advertisement in BLE Audio PTS tests.

The Broadcaster setup options was hardcoded to include the BT_LE_PER_ADV_OPT_USE_TX_POWER option. This causes incompabilities for any controller not support it, and since the option is not required for the audio tests it should not be used here.

### CODE REVIEW

**Review by Thalley — APPROVED** [review]

**Review by cvinayak — COMMENTED** [review]
LGTM... will wait on review from @mkasenberg or @sjanc

**Review by sjanc — APPROVED** [review]

**Review by cvinayak — APPROVED** [review]

**Comment by Frodevan** [comment]
@jhedberg Any updates on this?

**Comment by jhedberg** [comment]
> @jhedberg Any updates on this?

I had ignored it since it's audio related. Re-assigned to @Thalley now, so the merge criteria are satisfied.

**Pull request merged**
